### PR TITLE
Update base-nl.yaml

### PR DESCRIPTION
**Pull request by chris-kruining**
global.suffix.thousands :: should be uppercase as is stands for Kilo, which is shortened to K
global.infinite :: `oneindig` is the translation for infinite, not sure if keeping it as `inf` was on purpose.
global.time.* :: consistency
mainMenu.changelog :: -
mainMenu.continue :: technically a more correct translation
dialogs.buttons.restart :: consistency
dialogs.gameLoadFailure.title :: seems better to me

I will try to translate some more in the future, but my patience has run out at this moment :P

### CODE REVIEW

File: translations/base-nl.yaml [hunk 1]
```diff
@@ -97,23 +97,23 @@ global:
...
     suffix:
-        thousands: k
+        thousands: K
         millions: M
         billions: B
         trillions: T
 
     # Shown for infinitely big numbers
-    infinite: inf
+    infinite: oneindig
```
Comment (DerFeldspatz) [inline translations/base-nl.yaml:106]: This might be too long. Use a shortening or infinity sign

File: translations/base-nl.yaml [hunk 2]
```diff
@@ -97,23 +97,23 @@ global:
 
     # The suffix for large numbers, e.g. 1.3k, 400.2M, etc.
     suffix:
-        thousands: k
+        thousands: K
```
Comment (DerFeldspatz) [inline translations/base-nl.yaml:100]: You could also switch to SI (k,M,G,T) if there are no shortenings in dutch

Comment (chris-kruining) [reply translations/base-nl.yaml:100]: there is a shortening, and as I explained, should be an uppercase K, not lowercase

Comment (DerFeldspatz) [reply translations/base-nl.yaml:100]: Okay

File: translations/base-nl.yaml (hunk 1 above)
Comment (chris-kruining) [reply translations/base-nl.yaml:106]: as in the utf8 char for infinity? `∞` (U+221E)

Comment (DerFeldspatz) [reply translations/base-nl.yaml:106]: Yeah, you can use it if you want

**Comment by tobspr** [comment]
Could you have a look at the conflicts? :)

**Comment by chris-kruining** [comment]
@tobspr done

File: translations/base-nl.yaml [hunk 3]
```diff
@@ -117,8 +117,8 @@ dialogs:
         title: Savegame geïmporteerd
         text: Je savegame is succesvol geïmporteerd.
     gameLoadFailure:
-        title: Het spel is kapot
-        text: "Het laden van je savegame is mislukt:"
+        title: Corrupte savegame
+        text: Het laden van je savegame is mislukt:
```
Comment (ZrubekLukas) [inline translations/base-nl.yaml:121]: The quotes should stay at line 121 otherwise you get the lint-check error.

Comment (chris-kruining) [reply translations/base-nl.yaml]: done

**Comment by tobspr** [comment]
Ty!

**Pull request merged**